router.go

```diff
@@ -59,8 +59,8 @@ func (r *Router) Add(method, path string, h HandlerFunc) {
 	if path[0] != '/' {
 		path = "/" + path
 	}
-	ppath := path        // Pristine path
```

Comment:
Same as above and let's revert all the renaming for consistency.

Reply:
fixed